packages/terra-tabs/src/TabPane.jsx

```diff
@@ -40,12 +40,17 @@ const propTypes = {
    * If enabled, this prop will apply the `aria-selected` style to the pane.
    */
   isActive: PropTypes.bool,
+  /**
+   * If enabled, this prop will show the icon on the tab and also in the menu if pane is collapsed.
+   */
+  showIconInTabAndMenuWhenCollapsed: PropTypes.bool,
```

Comment:
Nit :  prop description is providing detailed explanation of prop usage so we can keep prop name short and simple.
```suggestion
  showIcon: PropTypes.bool,
```

Reply:
done:
https://github.com/cerner/terra-framework/pull/1592/commits/3fc0c686f4747389c60a7d563883b80ed4e799e6